server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/http/AuthenticationRoutes.java

```diff
@@ -98,9 +98,9 @@ public Logger logger() {
     @Override
     public HttpServerRoutes define(HttpServerRoutes builder) {
         return builder
-            .post(AUTHENTICATION, this::post)
-            .get(AUTHENTICATION, this::returnEndPointsResponse)
-            .delete(AUTHENTICATION, this::delete)
+            .post(AUTHENTICATION, JMAPRoutes.corsHeaders(this::post))
```

Comment:
could we find a better way to express that all requests should have CORS headers?
I don't like that call to be duplicated everywhere

Reply:
https://github.com/linagora/james-project/pull/3211 JMAP Y ports route registration in a centralized place (JMAPServer) where we could easily set up a unified CORS header definition.

I'll see if I suceed to hack something quickly but this will most likely be a follow up refactoring.